packages/pages/src/components/link/methods.test.ts

```diff
@@ -0,0 +1,97 @@
+import { getHref, isEmail, reverse } from "./methods";
+
+enum LinkType {
```

Comment:
Shouldn't this pull in from types instead of redefining?

Reply:
@bhainesva ?

Reply:
fixed